src/operator/lrn.cc

```diff
@@ -38,9 +38,26 @@ Operator* LocalResponseNormProp::CreateOperatorEx(Context ctx, std::vector<TShap
 DMLC_REGISTER_PARAMETER(LRNParam);
 
 MXNET_REGISTER_OP_PROPERTY(LRN, LocalResponseNormProp)
-.add_argument("data", "NDArray-or-Symbol", "Input data to the ConvolutionOp.")
+.add_argument("data", "NDArray-or-Symbol", "Input data.")
 .add_arguments(LRNParam::__FIELDS__())
-.describe("Apply convolution to input then add a bias.");
+.describe(R"code(Applies Local Response Normalization to the input.
```

Comment:
Let's not have this in capital letters. Just 

"Applies local response normalization to the input."

The main reasons to use capital letters would be if we are introducing a proper noun or an acronym.